sample-apps/react/react-video-demo/src/components/Views/LobbyView/LobbyView.module.css

```diff
@@ -36,19 +36,24 @@
 }
 
 .loadingPanel {
-  @apply bg-video-grey-600/5 p-2 sm:p-0 md:w-4/12 flex flex-col items-center;
+  @apply bg-video-grey-600/5 p-2 sm:p-0 md:w-4/12 flex flex-col items-center text-video-white;
 }
 
 .loading {
   @apply flex h-20 w-20;
-  animation: rotating 2s linear infinite;
+  animation: boat 1s linear infinite;
```

Comment:
This could also become a custom tailwind animation in case it will be used in other places across the code.

Reply:
Currently it's only used here. I don't have a proper design for the loading screen yet so I improvised this